fastai/callback/fp16.py

```diff
@@ -114,14 +134,22 @@ def after_fit (self): self.learn.model = convert_network(self.model, dtype=torch
 class NonNativeMixedPrecision(Callback):
     "Run training in mixed precision"
     order=10
-    def __init__(self, loss_scale=512, flat_master=False, dynamic=True, max_loss_scale=2.**24,
-                 div_factor=2., scale_wait=500, clip=None):
+    def __init__(self,
+        loss_scale:int=512, # Loss scale to use if not using dynamic
+        flat_master:bool=False, # Whether to flatten fp32 parameters for performance
+        dynamic:bool=True, # Whether to automatically determine loss scaling
+        max_loss_scale:float=2.**24, # Starting value for dynamic loss scaling
+        div_factor:float=2., # Divide by this on overflow, multiply by this after scale_wait batches
+        scale_wait:int=500, # Number of batches to wait for increasing loss scale
+        clip:float=None, # Value to clip gradients at, max_norm, as in `nn.utils.clip_grad_norm_`
+):
```

Comment:
```suggestion
    ):
```

Reply:
ugh, I probably spent a good 15 minutes staring at this wondering why jupyter was formatting it the way you want here, thinking jupyter was broken, mystery solved :)